src/App.js

```diff
@@ -89,15 +88,45 @@ class App extends React.Component {
             delete updateDataByStatuses[status];            
         }
         
-        this.setState({            
-            dataByStatuses: updateDataByStatuses
+        this.setState({       
+            dataCard: {
+                dataByStatuses: updateDataByStatuses,
+                statuses
+            },
+            modalWindow: this.state.modalWindow            
         });
     }
 
+    handleModalInfo = (card) => {
+        document.body.style.overflow = "hidden";
+        
+        this.setState({
+            dataCard: this.state.dataCard,
+            modalWindow: {
+                visible: true,
+                card
+            }
+        })        
+    }
+
+    handleCloseModal = () => {
+        document.body.style.overflow = "visible";
```

Comment:
it's not good to change DOM this way, instead you need to move state to the app and handle body class with state but let's leave it as is for now